docs-source/content/userguide/managing-fmw-domains/soa-suite/Logging-Exporter-Setup.md

```diff
@@ -0,0 +1,126 @@
+# Weblogic Logging Exporter Setup to publish logs into Elasticsearch
+
+The WebLogic Logging Exporter adds a log event handler to WebLogic Server. 
+
+WebLogic Server logs can be pushed into Elastic Stack in Kubernetes directly, by using the Elasticsearch REST API. For more details refer to [weblogic-logging-exporter](https://github.com/oracle/weblogic-logging-exporter) project.  
+
+This documents provides the steps to publish WebLogic Server logs into Elasticsearch and interact with them in Kibana. 
+
+Please follow the below steps to push the server server logs into Elasticsearch.  
+
+## Download Weblogic logging exporter binaries
+
+The Weblogic logging exporter pre-built binaries are available in the release page of the [weblogic-logging-exporter](https://github.com/oracle/weblogic-logging-exporter/releases) project.  
+You need to download the 
+* "weblogic-logging-exporter-0.1.1.jar" from this [link](https://github.com/oracle/weblogic-logging-exporter/releases)
+* "snakeyaml-1.23.jar" from [Maven Central](https://search.maven.org/artifact/org.yaml/snakeyaml/1.23/bundle).  
+
+## Copy JAR files to the Kubernetes Weblogic Domain Pod
+
+Copy the weblogic-logging-exporter-0.1.1.jar and snakeyaml-1.23.jar files to the domain home folder in the Admin server pod.
```

Comment:
all of this should be done as part of image creation, not after the fact

Reply:
After the fact could make sense for domain home on volume, but I agree that it's still awkward.  It would be better to make this part of domain creation.